src/tigerbeetle/benchmark_load.zig

```diff
@@ -276,11 +292,23 @@ const Benchmark = struct {
         while (b.transfer_index < b.transfer_count and
             b.batch_transfers.items.len < b.batch_transfers.capacity)
         {
-            const debit_account_index = random.uintLessThan(u64, b.account_count);
-            var credit_account_index = random.uintLessThan(u64, b.account_count);
-            if (debit_account_index == credit_account_index) {
-                credit_account_index = (credit_account_index + 1) % b.account_count;
-            }
+            const account_count_cold = b.account_count - b.account_count_hot;
+
+            const debit_account_hot = b.account_count_hot > 0 and
+                random.uintLessThan(u64, 100) < b.transfer_hot_percent;
+            const debit_account_index = if (debit_account_hot)
+                random.uintLessThan(u64, b.account_count_hot)
+            else
+                random.uintLessThan(u64, account_count_cold) + b.account_count_hot;
```

Comment:
```suggestion
            const debit_account_index = if (debit_account_hot)
                random.uintLessThan(u64, b.account_count_hot)
            else
                random.uintLessThan(u64, account_count);
```

Otherwise it'll have a counter-intuitive behavior where cold accounts are actually hot ones, when `b.account_count_hot` is "almost all accounts" and `b.transfer_hot_percent` is "very low". 

It also feels more intuitive that non-hot transfer selects account at random.